backend/Tasque.BLL/Services/OrganizationService.cs

```diff
@@ -27,6 +27,7 @@ public override async Task<OrganizationInfoDto> Create(OrganizationCreateDto cre
             entityToCreate.AuthorId = _currentUserId;
 
             await _dbSet.AddAsync(entityToCreate);
+            await _db.SaveChangesAsync();
```

Comment:
do we need to have 2 `.Save` calls in one method ?

Reply:
To get correct organizationId while creating organizationRole for currentUser